proto/viam/robot/v1/robot.proto

```diff
@@ -113,6 +113,13 @@ service RobotService {
       post: "/viam/api/v1/sessions/{id}/heartbeat"
     };
   }
+
+  // Log sends logs to be logged by this robot. Currently used for module logging.
+  rpc Log(LogRequest) returns (LogResponse) {
```

Comment:
I named the endpoint `Log` instead of `ModuleLog` in case other entities want to use it in the future.